apps/src/templates/page.html.ejs

```diff
@@ -78,6 +78,7 @@
 </div>
 
 <div id="visualizationResizeBar">
+  <i id="visualizationResizeIcon" class="fa fa-ellipsis-v"></i>
```

Comment:
Surprisingly to me to see the `<i>` tag here. Long debate here on whether or not it makes any sense: http://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span